### PR TITLE
[KOB-4317] - Update content for consuming endpoint `/devices` of Kobiton REST API

### PR DESCRIPTION
**Pull request by NguyenHoanTien**

### CODE REVIEW

**Comment by NguyenHoanTien** [comment]
This part are just update content for **get devices info**, and I just move the old part (get session info) into folder  **get session data**

File: kobiton-rest-api/get-devices-info/README.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+- Expected output for all samples in this section:
+```javascript
+{"privateDevices":[...],"favoriteDevices":[...],"cloudDevices":[...]}
+```
+
+### 1. Get all devices
+Basic command to get devices data from Kobiton REST API. This will get all devices and devices info that the current user can access. 
+
+**Smaple:**
```
Comment (haitranb) [inline kobiton-rest-api/get-devices-info/README.md]: Sample

File: kobiton-rest-api/get-devices-info/README.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+  -u <YOUR_KOBITON_USERNAME>:<YOUR_KOBITON_API_KEY>
+```
+
+**For user in org and group**
+```bash
+curl -X GET 'https://api.kobiton.com/v1/devices?isOnline=true&groupId=<YOUR_GROUP_ID>' \
+  -H 'Accept: application/json' \
+  -u <YOUR_KOBITON_USERNAME>:<YOUR_KOBITON_API_KEY>
+```
+
```
Comment (haitranb) [inline kobiton-rest-api/get-devices-info/README.md]: Add a note to show what happens when a user has a group but don't specify the groupID in the param? (Same for getSession and related section if required)

File: kobiton-rest-api/get-devices-info/README.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+### 3. Getting online Android/iOS devices
+Filter and get only online Android devices from all devices that current user can access. Adding Params:
+- `isOnline`: Check if the device is online **(True/False)**
+- `platformName`: The device platform **(Android/iOS)**
+
+**Smaple:**
```
Comment (haitranb) [inline kobiton-rest-api/get-devices-info/README.md]: Sample

File: kobiton-rest-api/get-devices-info/README.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+Filter and get specific devices, for this sample we will show to get all **Galaxy S8** devices with OS version **7.0**
+- `deviceName`: The device name, can use exact name or name, name, name.
+- `platformVersion`: The platform device version, can use exact version or version, version, version.
+- `appiumDisabled`: Check if devices can run automation test **(True/False)**
+
+> Note: 
+> - The devices name **'Galaxy S8'** should convert to **'Galaxy%20S8'** in the curl command. Since `%20` stand for blank space in curl command.
+> - This commands can do the same for any iOS devices, you just need to replace the value in `deviceName` param
+
+**Smaple:**
```
Comment (haitranb) [inline kobiton-rest-api/get-devices-info/README.md]: sample

File: kobiton-rest-api/get-devices-info/README.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+  - `<YOUR_KOBITON_USERNAME>`: Your Kobiton Username.
+  - `<YOUR_KOBITON_API_KEY>`: Your Kobiton API Key.
+  - `<YOUR_GROUP_ID>`: Your group Id
+
+- Expected output for all samples in this section:
+```javascript
+{"privateDevices":[...],"favoriteDevices":[...],"cloudDevices":[...]}
+```
+
+>**Note**: If you are in org and in groups but you do not specify param `groupId` when running below commands, Kobiton REST API will fetch all devices info in all group that you are assigned in.
```
Comment (haitranb) [inline kobiton-rest-api/get-devices-info/README.md]: If you are in an organization and groups but you do not specify the param `groupID` when running commands below, Kobiton REST API will fetch all devices that you can access, including devices in all groups that you were assigned and your private device (if any)

**Pull request merged**